frontend/app/component/form/polling_station/PollingStationForm.tsx

```diff
@@ -60,10 +67,21 @@ export function PollingStationForm({ electionId, pollingStation, onSaved, onCanc
     <div>
       {requestState.status === "api-error" && (
         <FormLayout.Alert>
-          <Alert type="error">{requestState.error.message}</Alert>
+          {requestState.error.reference === "EntryNotUnique" ? (
+            <Alert type="error">
+              <h2 id="testt">
```

Comment:
```suggestion
              <h2 id="testt">
```
This id needs to be updated to something non `testt`